index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="pt-br">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title> Final da Libertadores </title>
+</head>
+
+<body>
+    <article>
+        <header>
+            <h1>Final da Libertadores da America Fluminense X Boca Juniors (ARG) </h1>
+            <p>Postado por Alef Rodrigues</p>
+        </header>
+        
+        <nav>
+          <ul>
+            <li><a href="#">Introdução</a></li>
+            <li><a href="#">Desenvolvimento</a></li>
+            <li><a href="#">Conclusão</a></li>
+          </ul>
+        </nav>
+        
+        <section>
+            <h2>Introdução</h2>
+            <p>O Fluminense enfrentará o Boca Juniors (ARG), no próximo dia 4 de novembro, em busca do título da Copa
+                Libertadores.</p>
+        </section>
+
+        <section>
+            <h2>Desenvolvimento</h2>
+            <p>A partida acontecerá no Maracanã, local que o Tricolor tem um bom retrospecto jogando nessa competição.
+            </p>
+
+            <p>Ao todo foram 24 partidas disputadas pelo clube das Laranjeiras no estádio em confrontos pelo torneio
+                sul-americano, com 14 vitórias, 6 empates e apenas 4 derrotas. Além disso, foram 47 gols marcados e 23
+                sofridos. </p>
+
+        </section>
```

Comment:
Esse é um bom caso de exemplo, poderia ser reescrito da seguinte forma:

```suggestion
  <article>
            <h2>Desenvolvimento</h2>
                <section>
                  <h3>Local</h3>
                  <p>A partida acontecerá no Maracanã, local que o Tricolor tem um bom retrospecto jogando nessa competição.
                  </p>
        </section>
        <section>
            <h3>Histórico</h3>
            <p>Ao todo foram 24 partidas disputadas pelo clube das Laranjeiras no estádio em confrontos pelo torneio
                sul-americano, com 14 vitórias, 6 empates e apenas 4 derrotas. Além disso, foram 47 gols marcados e 23
                sofridos. </p>
        </section>
        </article>
```

Como os section também pedem um titulo, esse pode ser um formato válido.